pkg/util/util_test.go

```diff
@@ -0,0 +1,79 @@
+package util_test
+
+import (
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/drud/ddev/pkg/util"
+	"github.com/drud/ddev/pkg/version"
+	"github.com/drud/drud-go/utils/system"
+	docker "github.com/fsouza/go-dockerclient"
+)
+
+var (
+	TestArchiveURL        = "https://github.com/drud/wordpress/archive/v0.4.0.tar.gz"
```

Comment:
If these are going to be exported, they need a good comment with usage information.